vignettes/ggplot2.Rmd

```diff
@@ -0,0 +1,212 @@
+
+To use scales, one can use one of the scale functions that are patterned as `scale_{aesthetic}_{type}()` functions, where `{aesthetic}` is one of the pairings made in the mapping part of a plot. To map the `class` column in the `mpg` dataset to the viridis colour palette, we can write the following:
+
+```{r example_scales}
+#| fig.alt = "A scatterplot showing city versus highway miles per gallon for 
+#|  234 cars. The points are coloured according to seven classes of cars."
+ggplot(mpg, aes(cty, hwy, colour = class)) +
+  geom_point() +
+  scale_colour_viridis_d()
+```
+
+## Facets
+
+[Facets](https://ggplot2-book.org/facet.html) can be used to separate small multiples, or different subsets of the data. 
+It is a powerful tool to quickly split up the data into smaller panels to look for meaning among the subsets.
+In addition, the layout can convey a pattern because more than one variable may be used to split the data.
+
+The facets have their own mapping that can be given to the `vars()` function. 
+To show the `mpg` dataset by the `drv` and `year` columns, we can use `facet_grid()` as follows:
+
+```{r example_facets}
+#| fig.alt = "Scatterplot of city versus highway miles per gallon, for 234 cars.
+#|  The plot has six panels in a 2-row, 3-column layout, showing the 
+#|  combinations of three types of drive train and year of manifacture."
+ggplot(mpg, aes(cty, hwy)) +
+  geom_point() +
+  facet_grid(year ~ drv)
+```
+
+## Coordinates
+
+You can view the [coordinates](https://ggplot2-book.org/coord.html) part of the plot as an interpreter of position aesthetics. 
+While typically Cartesian coordinates are used, the coordinate system powers the display of [map](https://ggplot2-book.org/maps.html) projections and [polar](https://ggplot2-book.org/coord.html#polar-coordinates-with-coord_polar) plots.
+
+We can also use coordinates to display a plot with a fixed aspect ratio, so that one unit has the same length in both the x and y direction. The `coord_fixed()` function sets this ratio automatically.
+
+```{r example_coords}
+#| fig.alt = "A scatterplot showing city versus highway miles per gallon for 
+#|  234 cars. The aspect ratio of the plot is such that units on the x-axis
+#|  have the same length as units on the y-axis."
+ggplot(mpg, aes(cty, hwy)) +
+  geom_point() +
+  coord_fixed()
+```
+
+## Theme
+
+The [theme](https://ggplot2-book.org/polishing.html) system controls almost any visuals of the plot that are not controlled by the data and is therefore important for the look and feel of the plot. You can use the theme for alterations ranging from determining the location of the legends to setting the horizontal text adjustment for the bottom x-axis specifically. Many elements in the theme are hierarchical in that setting the look of the general axis line affects those of the bottom and left axes simultaneously.
```

Comment:
```suggestion
The [theme](https://ggplot2-book.org/polishing.html) system controls almost any visuals of the plot that are not controlled by the data and is therefore important for the look and feel of the plot. You can use the theme for customizations ranging from changing the location of the legends to setting the background color of the plot. Many elements in the theme are hierarchical in that setting the look of the general axis line affects those of the x and y axes simultaneously.
```

Reply:
Changed the example to make it more accessible to someone who hasn't made ggplots before.